### PR TITLE
fix: test-recording instructions in Component Test mode

### PR DESCRIPTION
**Pull request by marktnoonan**
- Closes https://cypress-io.atlassian.net/browse/UNIFY-1637

### User facing changelog

In component testing mode, when no runs have been recorded to the dashboard, the command to copy will now include the `--component` flag, so that we run in component testing mode.

Additionally, the project root has been removed from the terminal prompt in all situations (per Brian's comment in this ticket). Terminal prompts now begin with `$` always, instead of `some-project: $`.

### How has the user experience changed?

Before - a command is shown in component testing mode, but this command would run cypress in the default e2e mode, thus not record component tests:
Screen Shot 2022-05-10 at 6 01 56 PM

After - correct command is shown for component testing (it includes the `--component` flag:
Screen Shot 2022-05-10 at 5 59 46 PM

### How to test manually

Set up a project with no tests recorded in the dashboard and visit the runs page in the app, it should invite you to record your first run, and recommend the correct command for your testing type.

### PR Tasks

- [x] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/app/cypress/e2e/runs.cy.ts [hunk 1]
```diff
@@ -1,6 +1,35 @@
 import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
 import type { SinonStub } from 'sinon'
 
+function scaffoldTestingTypeAndVisitRunsPage (testingType: 'e2e' | 'component') {
+  if (testingType === 'component') {
+    cy.scaffoldProject('component-tests')
```
Comment (ZachJW34) [inline packages/app/cypress/e2e/runs.cy.ts:6]: Can we use the `cypress-in-cypress` project? It has both testing types configured so we wouldn't have to create another system test project.

File: packages/app/src/runs/RunsEmpty.vue [hunk 2]
```diff
@@ -58,7 +59,9 @@ const firstRecordKey = computed(() => {
     : '<record-key>'
 })
 const recordCommand = computed(() => {
-  return `cypress run --record --key ${firstRecordKey.value}`
+  const componentFlagOrSpace = props.gql.currentTestingType === 'component' ? ' --component ' : ' '
```
Comment (ZachJW34) [inline packages/app/src/runs/RunsEmpty.vue:62]: You could do a `['arg1', 'arg2'].join(' ')` here if you wanted to to remove the space addition.

**Review by ZachJW34 — APPROVED** [review]
Few minor comments, not blocking

**Review by tbiethman — APPROVED** [review]
Code and percy snapshots look good 👍

File: packages/app/cypress/e2e/runs.cy.ts (hunk 1 above)
Comment (marktnoonan) [reply packages/app/cypress/e2e/runs.cy.ts]: will merge when CI goes green

**Pull request merged**